packages/@react-aria/datepicker/src/useDatePickerGroup.ts

```diff
@@ -31,7 +33,21 @@ export function useDatePickerGroup(state: DatePickerState | DateRangePickerState
         e.preventDefault();
         e.stopPropagation();
         if (direction === 'rtl') {
-          focusManager.focusNext();
+          if (orderedSegments) {
+            let button = ref.current?.querySelector('button');
```

Comment:
is it possible someone could add another button to the Group? such as in RAC?
If so, it's be best to find the element by it's `id` if we can get that or some other more unique attribute

Reply:
yeah you're right. https://github.com/adobe/react-spectrum/issues/7645 is a good case of how someone might add buttons in either through hooks or rac. if that's the case, maybe we should also query select all buttons? i'm assuming that they should be able to keyboard to any type of button so long as it's within the datefield?